.github/workflows/ci_cron.yml

```diff
@@ -38,9 +38,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Install required packages
```

Comment:
Have you forced this ci_cron to be run yet? Since it doesn't run on PRs, I'm worried that there could be failures after merging from changing this that we haven't seen yet. If you haven't yet, maybe adding a PR trigger to here temporarily would give us a better idea of how this will affect all the ruby versions

Reply:
Good call. Originally I was only changing the initial `apt-get` call for the unit tests. I went ahead and updated the `apt-get` call for the multiverse tests as well, and got everything (ci_cron included) working with #1365. I have updated this PR's description to accurately reflect everything that changed. We ended up being able to remove the packages needed to compile Ruby from source, but had to retain 3 packages for instrumented gems.